catalogue/models/messages.py

```diff
@@ -18,12 +28,28 @@ class Meta:
         null=True,
         verbose_name=Component._meta.verbose_name,
     )
-    created = models.DateTimeField("Erstellt", auto_now_add=True)
-    name = models.CharField("Name", max_length=50)
-    mail = models.EmailField("E-Mail-Adresse")
+
     message = models.TextField(
         "Nachricht", help_text="Ihre Nachricht an den Anbieter", max_length=2000
     )
 
     def __str__(self):
         return "{} {}".format(self._meta.verbose_name, self.id)
+
+
+
+class Feedback(SenderInfo):
+    class Meta:
+        verbose_name = "Feedback"
+        verbose_name_plural = "Feedbacks"
+
+    CHOICES=[('positive', 'Positiv'),
+         ('neutral', 'Neutral'),
+         ('negativ', 'Negativ')]
+    sentiment = models.CharField("Wie zufieden sind Sie mit der Suche?", max_length=8, choices=CHOICES, default=CHOICES[0][0])
+    message = models.TextField(
+        "Nachricht", help_text="Ihr Feedback", max_length=2000
+    )
+    search_url = models.URLField(default=None, max_length=200)
```

Comment:
can easily be longer than 200 chars after user selects some facets. better to use an unlimited textfield